test/load_tests.jl

```diff
@@ -0,0 +1,13 @@
+using Revise
```

Comment:
Seems like even if the best practice is to install Revise globally, if our tests are going to depend it we ought to have it in `test/Project.toml`

Reply:
This was an oversight. We shouldn't add Revise to the test env

Reply:
Revise definitely gets used in this PR though

Reply:
The default run script for testing in CI doesn't use Revise